test/components/interactive/dragBoxLayerTests.ts

```diff
@@ -50,6 +50,15 @@ describe("Interactive Components", () => {
       svg.remove();
     });
 
+    it("clipPath enabled", () => {
+      /*
+       * clipPath has to be enabled so the resize handle doesn't show on hovering
+       * over the edge of the drag box
+       */
```

Comment:
In regards to this comment, is this mainly to avoid having the drag box spill over the edge of the cell?  What's the context of this change?

Can you provide context in the pull request description?